### PR TITLE
INT-2091: Paycore/Cardgate: Add new system currencies

### PR DESCRIPTION
**Pull request by yuliia-bondareva-corefy**

### CODE REVIEW

File: data/payment_services.json [hunk 1]
```diff
@@ -16596,5 +16596,23 @@
...
     "amount_min": 0.01,
     "amount_max": 100000
+  },
+  {
+    "status": "active",
+    "code": "payment_card_bhd_hpp",
+    "flow": "hpp",
+    "method": "payment_card",
+    "currency": "BHD",
+    "amount_min": 0.001,
```
Comment (TonyTramb) [inline data/payment_services.json:16606]: 0.03
эквивалент 0.1 USD

File: data/payment_services.json [hunk 2]
```diff
@@ -16596,5 +16596,23 @@
...
+    "amount_max": 1000000
+  },
+  {
+    "status": "active",
+    "code": "payment_card_omr_hpp",
+    "flow": "hpp",
+    "method": "payment_card",
+    "currency": "OMR",
+    "amount_min": 0.001,
```
Comment (TonyTramb) [inline data/payment_services.json:16615]: 0.03
эквивалент 0.1 USD

File: data/payment_services.json [hunk 3]
```diff
@@ -16596,5 +16596,23 @@
...
+    "amount_max": 1000000
```
Comment (TonyTramb) [inline data/payment_services.json:16616]: 40000
эквивалент 100.000 USD

File: data/payment_services.json [hunk 4]
```diff
@@ -16596,5 +16596,23 @@
...
     "amount_max": 100000
+  },
+  {
+    "status": "active",
+    "code": "payment_card_bhd_hpp",
+    "flow": "hpp",
+    "method": "payment_card",
+    "currency": "BHD",
+    "amount_min": 0.001,
+    "amount_max": 1000000
```
Comment (TonyTramb) [inline data/payment_services.json:16607]: 40000
эквивалент 100.000 USD

File: data/payment_services.json (hunk 1 above)
Comment (yuliia-bondareva-corefy) [reply data/payment_services.json]: fixed

File: data/payment_services.json (hunk 2 above)
Comment (yuliia-bondareva-corefy) [reply data/payment_services.json]: fixed

File: data/payment_services.json (hunk 3 above)
Comment (yuliia-bondareva-corefy) [reply data/payment_services.json]: fixed

File: data/payment_services.json (hunk 4 above)
Comment (yuliia-bondareva-corefy) [reply data/payment_services.json]: fixed

**Review by TonyTramb — APPROVED** [review]

**Pull request merged**